### ISSUE TITLE
Environment link is broken

### ISSUE THREAD

**Issue by patricia-ternes** [datacarpentry/python-ecology-lesson#574]
The link to download the environment file in the [Summary and Setup](https://datacarpentry.org/python-ecology-lesson/#alternative-installing-required-packages-with-environment-file) is not working.

The problem is in the line 196 of the file `learners/setup.md` that can be access via this [permalink](https://github.com/datacarpentry/python-ecology-lesson/blob/fbb0e211d0f795917af80144c3ffde5d1680639e/learners/setup.md?plain=1#L196C1-L196C74)

I believe the problem can be solved using the file path, something like:

```markdown
[environment.yml](episodes/files/environment.yml)
```

The complete link should be generated automatically when deploy the material.

**Comment by btovar** [comment]
@patricia-ternes Thanks for your report!  Would you like to submit a pr to fix this issue?

**Comment by patricia-ternes** [comment]
Yes, can you confirm if the path is the way to solve? I can do this now!

**Comment by btovar** [comment]
The url that works is:  `https://datacarpentry.org/python-ecology-lesson/files/environment.yml`
So I think we need:  `[environment.yml](/files/environment.yml)`, since `episodes` becomes the root of the pages.

**Comment by btovar** [comment]
@patricia-ternes Thank you for you help with this issue!

**Comment by zkamvar** [comment]
Hello @btovar and @patricia-ternes, thank you for helping out with this fix and I apologise about the sparsity of the documentation about relative links. I've updated the workbench documentation to be a bit more clear about them: https://carpentries.github.io/sandpaper-docs/episodes.html#internal-links 

> The url that works is: `https://datacarpentry.org/python-ecology-lesson/files/environment.yml` So I think we need: `[environment.yml](/files/environment.yml)`, since `episodes` becomes the root of the pages.

This is _almost_ right with respect to thinking about relative paths, but in terms of working with The Workbench, since this is referenced from the `learners/setup.md` page, you will want to point to the file in the project _relative to_ the page: `[environment.yml](../episodes/files/environment.yml)`.

**Comment by btovar** [comment]
Thanks @zkamvar, that makes sense. I wasn't sure if the "episodes" part could be refer to after it becomes the root.
